jsonschema-maven-plugin/README.md

```diff
@@ -0,0 +1,133 @@
+# Java JSON Schema Generator – Maven Plugin
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.victools/jsonschema-module-javax-validation/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.victools/jsonschema-maven-plugin)
+
+Maven plugin for the [jsonschema-generator](../jsonschema-generator) – Integrating JSON Schema generation in your builds
+
+## Features
+1. Generate the JSON schema for a Java class.
+2. Configure the generator by selecting the modules to be used.
+3. Configure the options of the modules used in the generation.
+4. Configure the use of custom modules.
+
+## Usage
+### Plugin definition
+```xml
+<plugin>
+    <groupId>com.github.victools</groupId>
+    <artifactId>jsonschema-maven-plugin</artifactId>
+    <version>4.8.0</version>
+    <executions>
+        <execution>
+            <goals>
+                <goal>schema-generator</goal>
+            </goals>
+        </execution>
+    </executions>
+    <configuration>
+        <className>com.myOrg.myApp.myclass</className>
+    </configuration>
+</plugin>
+```
+This will use the default configuration of the generator. This will enable all of the modules that are part of the generator. 
+
+
+### Configuring file locations
+```xml
+<configuration>
+    <className>com.myOrg.myApp.myclass</className>
```

Comment:
Let's follow standard naming conventions also in those examples (same applies for the others below). 😄
```suggestion
    <className>com.myOrg.myApp.MyClass</className>
```

Reply:
ok